server/routes/fish.js

```diff
@@ -17,6 +17,12 @@ router.get('/:id', (req, res) => {
     .then(fishDetail => res.status(200).json(fishDetail))
 })
 
+router.put('/:id', (req, res) => {
+  const fishId = Number(req.params.id)
+  fishDetails.where('fishId', fishId)
+    .update(fishDetails => res.status(200).json(fishDetails))
+})
```

Comment:
How about using your db function you created in `server/db/fish` to not duplicate the code and then have the `res.status...` here?

Reply:
This would look something more like:
```
router.put('/:id', (req, res) => {
  db.editItem(updatedDetails, id)
    .then(() => {
      res.status(202).json({
        ok: true,
        message: 'Successfully created.',
      })
    })
  .catch(() => {
      res.status(400).send({
        errorType: 'DATABASE_ERROR'
      })
    })
})
```